EMS/core-bundle/src/Service/Mapping.php

```diff
@@ -53,6 +61,7 @@ public function generateMapping(ContentType $contentType): array
     {
         $out = [
             'properties' => [],
+            'dynamic' => 'false' === $this->dynamicMapping,
```

Comment:
Possible values are true, false, strict and runtime.

If we define EMSCO_DYNAMIC_MAPPING='runtime'  -> it will just set dynamic to true ?

Reply:
Oops, fixed..